tests/models/test_unit_hostfactory.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+
+from conjur_api.models import CreateHostData
+
+
+class HostFactoryTest(unittest.TestCase):
+
+    def test_create_host_data_fields(self):
```

Comment:
Can we please add more tests? I don't think this single test covers the functionality that you added like:

- Creating a host and seeing that the host was created successfully with annotations
- Failing to create a host
- Adding multiple annotations to ensure that the dictionary translation works with multiple keys and values

Also can you please also add these same tests as integration tests as well?

Reply:
I will test the scenarios you described in unit and integration tests